plugins/aws/fix_plugin_aws/resource/bedrock.py

```diff
@@ -549,7 +551,7 @@ class AwsBedrockModelCustomizationJob(BedrockTaggable, BaseAIJob, AwsResource):
         "bedrock",
         "list-model-customization-jobs",
         "modelCustomizationJobSummaries",
-        expected_errors=["ValidationException"],
+        expected_errors=["ValidationException", "InternalServerException"],
```

Comment:
InternalServerException: sounds like something that should be retried and not ignored?